src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -124,21 +124,150 @@ class PerformanceLattice(BaseModel):
+            spin-echo dynamical decoupling protocol. This measurement isolates the effects of incoherent processes (measured in s)
+        T2EchoEnsemble (Decimal): Dephasing time of an ensemble of non-interacting qubits distributed over the user region, in the absense
+            of drive, as measured by a spin-echo dynamical decoupling protocol. This measurement isolates the effects
+            of incoherent processes. (measured in s)
+        T2RabiSingle (Decimal): Typical decoherence time of a single driven qubit, as measured by a Rabi oscillation protocol with variable
+            pulse duration a maximum Rabi frequency. Includes coherent and incoherent processes.(measured in s)
+        T2RabiEnsemble (Decimal): Decoherence time of an ensemble of non-interacting driven qubits distributed over the user region, as measured
+            by a Rabi oscillation protocol with variable pulse duration at maximum Rabi frequency. Includes
+            coherent and incoherent processes. (measured in s)
+        T2BlockadedRabiSingle (Decimal): Typical decoherence time of a single pair of driven blockaded qubits, as measured by a Rabi oscillation
+            protocol with variable pulse duration a maximum Rabi frequency. Includes coherent and incoherent processes. (measured in s)
+        T2BlockadedRabiEnsemble (Decimal): Decoherence time of an ensemble of pairs of driven blockaded qubits distributed over the user region
+            (different pairs do not interact with each other), as measured by a Rabi oscillation protocol with variable pulse duration at
+            maximum Rabi frequency. Includes coherent and incoherent processes. (measured in s)
+        detuningError (Decimal): Systematic error from specified value of the global detuning averaged over the user
+            region. (measured in rad/s)
+        detuningInhomogeneity (Decimal): RMS inhomogeneity of the detuning over the user region. (measured in rad/s)
+        rabiAmplitudeRampCorrection (List[RabiCorrection]): dynamic correction curve of effective single-qubit
+            on-resonant Rabi oscillation frequency driven by a triangular amplitude waveform, relative
     """
 
     rabiFrequencyErrorRel: Decimal
+    rabiFrequencyGlobalErrorRel: Decimal
+    rabiFrequencyInhomogeneityRel: Decimal
+    groundDetectionError: Decimal
+    rydbergDetectionError: Decimal
+    groundPrepError: Decimal
+    rydbergPrepErrorBest: Decimal
+    rydbergPrepErrorWorst: Decimal
+    T1Single: Decimal
+    T1Ensemble: Decimal
+    T2StarSingle: Decimal
+    T2StarEnsemble: Decimal
+    T2EchoSingle: Decimal
+    T2EchoEnsemble: Decimal
+    T2RabiSingle: Decimal
+    T2RabiEnsemble: Decimal
+    T2BlockadedRabiSingle: Decimal
+    T2BlockadedRabiEnsemble: Decimal
+    detuningError: Decimal
+    detuningInhomogeneity: Decimal
+    rabiAmplitudeRampCorrection: [RabiCorrection]
```

Comment:
```suggestion
    rabiAmplitudeRampCorrection: list[RabiCorrection]
```

Reply:
I found we already majorly used List in this repository: example https://github.com/amazon-braket/amazon-braket-schemas-python/blob/2d38f86867f95194196134e463c71d340a7d8c7a/src/braket/ir/openqasm/modifiers.py#L37. So looks like List is also good here?

Reply:
I have edit the suggestion to go back to using `List` from typing and will open a separate PR that addresses this change for the whole repo. @guomanmin , you may still need to change [L15](https://github.com/amazon-braket/amazon-braket-schemas-python/pull/167/files/50863035e1f0a32f91e0e80773a26aec2d8ef94b#diff-ecdd6f26bdf146078b4f4dbcc54c0f81e04b8f9d0e143d37243915def886eadeR15) to:
```
from typing import List, Tuple
```

Reply:
Reverted back to using List and imported List from typing